src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/ServiceHearingPartiesMapping.java

```diff
@@ -90,7 +91,8 @@ public static PartyDetails createHearingPartyDetails(Entity entity, HearingOptio
         partyDetails.partyRole(HearingsPartiesMapping.getPartyRole(entity));
         partyDetails.partyName(HearingsPartiesMapping.getIndividualFullName(entity));
         partyDetails.individualDetails(getPartyIndividualDetails(entity, hearingOptions, hearingSubtype, partyId, refData));
-        partyDetails.partyChannel(HearingChannelUtil.getIndividualPreferredHearingChannel(hearingSubtype, hearingOptions, null).getHmcReference());
+        HearingChannel partyChannel = HearingChannelUtil.getIndividualPreferredHearingChannel(hearingSubtype, hearingOptions, null);
+        partyDetails.partyChannel(partyChannel != null ? partyChannel.getHmcReference() : null);
```

Comment:
I noticed this line does not have tests on the partyChannel = null condition
please see the build's sonar coverage, and see if you can incr the scrore